internal/rbd/rbd_util.go

```diff
@@ -81,6 +81,35 @@ const (
 
 	// clusterNameKey cluster Key, set on RBD image.
 	clusterNameKey = "csi.ceph.com/cluster/name"
+
+	// Qos parameters name of StorageClass.
+	baseReadIOPS            = "BaseReadIOPS"
+	baseWriteIOPS           = "BaseWriteIOPS"
+	baseReadBytesPerSecond  = "BaseReadBytesPerSecond"
+	baseWriteBytesPerSecond = "BaseWriteBytesPerSecond"
+	readIopsPerGB           = "ReadIopsPerGB"
```

Comment:
some variables and values have **IOPS** (my preference), some have **Iops**. Please use one format everywhere.

Reply:
Unified use `Iops` format